triangle/src.txt

```diff
@@ -0,0 +1,2 @@
+0 0 0 3 4 g
```

Comment:
Your pull request: TRIANGLE-1022 Added triangle.py


Your commit: LEETCODE-1022 Added triangle.py

Your commit: TRIANGLE-1022 Refactored triangle.py
Action of your comment must be from ['Added', 'Deleted']
Your commit: TRIANGLE-1022 Refactored triangle.py
Action of your comment must be from ['Added', 'Deleted']
Your commit: TRIANGLE-1022 Refactored triangle.py
Action of your comment must be from ['Added', 'Deleted']
Your commit: TRIANGLE-1022 Refactored triangle.py
Action of your comment must be from ['Added', 'Deleted']
Your commit: TRIANGLE-1022 Refactored triangle.py
Action of your comment must be from ['Added', 'Deleted']